src/src.ruleset

```diff
@@ -51,5 +51,17 @@
   </Rules>
   <Rules AnalyzerId="xunit.analyzers" RuleNamespace="xunit.analyzers">
     <Rule Id="xUnit1004" Action="None" />
+    <Rule Id="xUnit2013" Action="Hidden" />
```

Comment:
What's the difference between hidden and none? What are these rules you're disabling ? :)

Reply:
The difference is that with `hidden`, analysis is still performed, while with `none`, it is disabled. These should all have been `none`.

These are xUnit analyzers that I have disabled in the meantime since they're impacting existing tests, I wanted to avoid diverging from 1.x if not needed, but I'll re-enable these, make the fixes and port the changes.